import-export-cli/Petstore/Meta-information/api.yaml

```diff
@@ -0,0 +1,21 @@
+id:
```

Comment:
Seems the Petstore project is mistakenly committed. Shall we remove this folder?